images-list

```diff
@@ -1161,7 +1161,7 @@ quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.59.0
 quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.63.2
 quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.66.0
 quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.67.0
-quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.72.0
+quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.74.0
```

Comment:
See the PR checklist - you CANNOT remove existing tags. Add the new tag, without removing another one.

Reply:
Done